tests/test_app_integration.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+from unittest import mock
+from dotenv import load_dotenv, find_dotenv
+import app
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version 
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
```

Comment:
Having a .env.test file is correct but it actually isn't being used at all. Your test is using the real .env file (which is presumably why your test missing mocks still passes)

Your Config class is defining class variables - so they are assigned once when the class is defined. You can change the environment all you want afterwards, but Config.TRELLO_BASE_URL is already set.
As an aside, the class name is Config but the file name is trello_config. They should match. 

You could fix it by making them instance variables but really it would be nicer for the config class to have a setter method and then the correct values can be passed in when the app is created.

Reply:
Fixed. No idea how this was working in VS Code before I checked in the code (something iffy with my env in that dir). 
I checkout out to a clean directory and fix the errors